Zend/tests/match/044.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Match expression can be used as a constant expression unless parts aren't constant
+--FILE--
+<?php
+
+// Here, this test uses strtolower because the locale dependence and change to the string
```

Comment:
Incorrect comment :)

Reply:
Oops, got sidetracked checking if constant expressions were proposed before and forgot to update this.